services/Directory/data/FilterListViewUrl.json

```diff
@@ -15765,5 +15759,161 @@
+  },
+  {
+    "filterListId": 2551,
+    "id": 2771,
+    "primariness": 1,
+    "url": "https://github.com/stamparm/maltrail/blob/master/trails/static/suspicious/i2p.txt"
+  },
+  {
+    "filterListId": 2552,
+    "id": 2772,
+    "primariness": 1,
+    "url": "https://github.com/stamparm/maltrail/blob/master/trails/static/suspicious/ipinfo.txt"
+  },
+  {
+    "filterListId": 2553,
+    "id": 2773,
+    "primariness": 1,
+    "url": "https://github.com/stamparm/maltrail/blob/master/trails/static/suspicious/parking_site.txt"
+  },
+  {
+    "filterListId": 2554,
+    "id": 2774,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/DeadServerGamePingList.txt"
+  },
+  {
+    "filterListId": 2554,
+    "id": 2775,
+    "primariness": 2,
+    "url": "https://gitlab.com/DandelionSprout/adfilt/-/raw/master/DeadServerGamePingList.txt"
+  },
+  {
+    "filterListId": 2555,
+    "id": 2776,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Sensitive%20lists/TabloidRemover-MastodonCategoryForImports.csv"
+  },
+  {
+    "filterListId": 2555,
+    "id": 2777,
+    "primariness": 2,
+    "url": "https://gitlab.com/DandelionSprout/adfilt/-/raw/master/Sensitive%20lists/TabloidRemover-MastodonCategoryForImports.csv"
+  },
+  {
+    "filterListId": 2556,
+    "id": 2778,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/ClearURLs%20for%20uBo/clear_urls_uboified.txt"
+  },
+  {
+    "filterListId": 2556,
+    "id": 2779,
+    "primariness": 2,
+    "url": "https://gitlab.com/DandelionSprout/adfilt/-/raw/master/ClearURLs%20for%20uBo/clear_urls_uboified.txt"
+  },
+  {
+    "filterListId": 2557,
+    "id": 2780,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/scafroglia93/blocklists/master/blocklists-malware-traffic.txt"
+  },
+  {
+    "filterListId": 2558,
+    "id": 2781,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/Knuschberkeks/Knuschberblock/main/Knuschberblock.txt"
+  },
+  {
+    "filterListId": 2559,
+    "id": 2782,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/huzunluartemis/TurkishAdblockList/main/src/HostsList.txt"
+  },
+  {
+    "filterListId": 2560,
+    "id": 2783,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/huzunluartemis/TurkishAdblockList/main/src/ElementalList.txt"
+  },
+  {
+    "filterListId": 2561,
+    "id": 2784,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/huzunluartemis/TurkishAdblockList/main/src/BadIpList.txt"
+  },
+  {
+    "filterListId": 2562,
+    "id": 2785,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/furkun/AndroidSecurityHosts/main/hosts"
+  },
+  {
+    "filterListId": 2563,
+    "id": 2786,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/furkun/AndroidSecurityHosts/main/domains.txt"
+  },
+  {
```

Comment:
maybe this entry is a duplicate from copy/pasting?

https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=7099&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=ebf69b98-7e9b-5a4b-1523-fc9437d68b9b

Reply:
I think I see the problem only now (Was a bit busy with PC gaming): The `codeberg.org` below should be primariness 3, not 2. Give me a couple minutes.